src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/CompiledRegexRunnerFactory.cs

```diff
@@ -7,25 +7,21 @@ namespace System.Text.RegularExpressions
 {
     internal sealed class CompiledRegexRunnerFactory : RegexRunnerFactory
     {
-        private readonly DynamicMethod _goMethod;
-        private readonly DynamicMethod _findFirstCharMethod;
+        private readonly DynamicMethod _scanMethod;
         private readonly int _trackcount;
```

Comment:
As with the source generator, it'd be nice to see whether we can just delete _trackcount, the setting of it, etc.  That'll contribute to https://github.com/dotnet/runtime/issues/62450.